pkg/endpoint/provisioning.go

```diff
@@ -0,0 +1,41 @@
+package endpoint
+
+import "time"
+
+type ProvisioningRequest struct {
+	CertificateId *string
```

Comment:
`pickupId` should be `pickupID`. 
The same goes for the other attributes. Golang capitalizes the ID word. 
https://google.github.io/styleguide/go/decisions#initialisms

Reply:
Resolved by latest commit